fiftyone/utils/beam.py

```
@@ -5,13 +5,21 @@
 | `voxel51.com <https://voxel51.com/>`_
 |
 """
+import inspect
```

Ruff (0.11.9)

`inspect` imported but unused

Remove unused import: `inspect`

(F401)